com.unity.render-pipelines.high-definition/Runtime/Sky/HDRISky/HDRISky.cs

```diff
@@ -150,8 +150,8 @@ public override int GetHashCode()
                 hash = hash * 23 + dirLightShadow.overrideState.GetHashCode();
                 hash = hash * 23 + rectLightShadow.overrideState.GetHashCode();
 #else
-                hash = hdriSky.value != null ? hash * 23 + CoreUtils.GetTextureHash(hdriSky.value) : hash;
-                hash = flowmap.value != null ? hash * 23 + CoreUtils.GetTextureHash(flowmap.value) : hash;
+                hash = hdriSky.value != null ? hash * 23 + hdriSky.GetHashCode() : hash;
```

Comment:
we don't need the check for `hdriSky.value != null`. It's a corner case but even if texture is null, we should hash the override state. texture hash already contains the check for null so you can just do 
               ` hash = hash * 23 + hdriSky.GetHashCode();`
and same for flowmap

otherwise looks good

Reply:
Yeah, I don't know why it was like that